src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+$backroundColor: #eee;
```

Comment:
[CHECKLIST] 3.[BEM] - Create a separate file per each BEM block styles that have the same name as the block
Could you please create separate files for variables, body and calendar styles and add them to main.scss with @import